sdk/program/src/program_error.rs

```diff
@@ -43,6 +43,8 @@ pub enum ProgramError {
     BorshIoError(String),
     #[error("An account does not have enough lamports to be rent-exempt")]
     AccountNotRentExempt,
+    #[error("Unsupported operations")]
+    Unsupported,
```

Comment:
Specializing this error seems nicer: perhaps `UnsupportedSysvar`

Reply:
Yeah, I was on the fence about that, making a specialized error for every use case seemed messy for a case where unsupported can be a pretty obvious generic response, like in this case where you call a function that takes no arguments and it returns unsupported

Reply:
Switched to `UnsupportedSysvar`